assembly-combined-package/bin/install.sh

```diff
@@ -67,6 +68,42 @@ until mysql -h$MYSQL_HOST -P$MYSQL_PORT -u$MYSQL_USER -p$MYSQL_PASSWORD  -e ";"
      exit 1
 done
 
+
+########################  init LINKIS related env  ################################
+if [ "$LINKIS_HOME" = "" ]
+then
+  export LINKIS_HOME=${workDir}/LinkisInstall
+fi
+
+if [  -d $LINKIS_HOME ] && [ "$LINKIS_HOME" != "$workDir" ];then
+   echo "LINKIS_HOME: $LINKIS_HOME is alread exists and will be backed up"
+   if [  -d $LINKIS_HOME-bak ];then
+    rm -r $LINKIS_HOME-bak
```

Comment:
Would it be better to add a time suffix to the backup folder?
I think during the installation process, especially in the upgrade scenario, the install.sh script may be executed repeatedly, resulting in the loss of the original backup file。

Reply:
yeah, that's a good idea. I will fix it.

Actually, I just move the code snippet place from below.